client/src/App.tsx

```diff
@@ -14,23 +15,33 @@ function App() {
   // Current (pending) test runs
   const [testList, setTestList] = useState<TestResultSet[]>([]);
   const [busy, setBusy] = useState<boolean>(false);
+  const [wrongNetworkError, setWrongNetworkError] = useState<String | null>(
+    null,
+  );
   const [domain, setDomain] = useState("testanchor.stellar.org");
   const [currency, setCurrency] = useState<string>("");
   const [runOptionalTests, setRunOptionalTests] = useState<boolean>(
     Boolean(parseInt(process.env.RUN_OPTIONAL_TESTS || "0")) || false,
   );
+  const [runOnMainnet, setRunOnMainnet] = useState<boolean>(
+    Boolean(parseInt(process.env.MAINNET || "0")) || false,
```

Comment:
Does the app frontend have access to process.env?

Reply:
Maybe its different in production but in development the server and client are launched from the same parent process so I think so.